mlflow/types/utils.py

```diff
@@ -286,3 +303,22 @@ def _is_spark_df(x) -> bool:
         return isinstance(x, pyspark.sql.dataframe.DataFrame)
     except ImportError:
         return False
+
+
+def _validate_input_dictionary_contains_only_strings_and_lists_of_strings(data) -> bool:
+    invalid_keys = []
+    for key, value in data.items():
+        if isinstance(value, list):
+            if not any(isinstance(item, str) for item in value):
```

Comment:
```suggestion
        if isinstance(value, list) and not any(isinstance(item, str) for item in value):
```

Reply:
Oh yes, great catch. I'm adding coverage for every change that I'm making to `tests/types/test_schema.py` to ensure that we've got the bases covered here :)